src/components/Groups.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div >
+        <h1> Grupper i F-spexet </h1>
+        <p class="bparagraph"> Lorem ipsum dolar lel </p> 
+            <h2>Styret</h2>
+            <p> 
+            Styret är de som ansvarar för att det överhuvudtaget blir ett F-spex. De sitter i otaliga lunchmöten för att planera, boka, styra, och se till att spexet fungerar och är roligt.
+            </p>
+
+                <h3> Orförande</h3>
```

Comment:
:thinking:

Reply:
Please explain, content copy pasted from old site.

Reply:
Or**d**förande

Reply:
```suggestion
                <h3> Ordförande</h3>
```

Reply:
The one thing I wrote before I started copy paste. I'm impressed with me. Language is hard.

Reply:
"impressed with _**myself**_." thank you very much.